ssl/t1_lib.c

```diff
@@ -2780,16 +2775,23 @@ int ssl_parse_serverhello_tlsext(SSL *s, PACKET *pkt)
     return 1;
 }
 
+static RAW_EXTENSION *get_extension_by_type(RAW_EXTENSION *exts, size_t numexts,
+                                            unsigned int type)
+{
+    size_t loop;
+
+    for (loop = 0; loop < numexts; loop++) {
```

Comment:
because it's better than foo[bar].field repeated calculations -- not just for efficiency, but for mental understanding.  and it's also more future-proof and more clear.